content/src/Environment.ts

```diff
@@ -296,7 +297,11 @@ export class EnvironmentBuilder {
           ? DEFAULT_BLOCKS_SUBGRAPH_MATIC_MAINNET
           : DEFAULT_BLOCKS_SUBGRAPH_MATIC_MUMBAI)
     )
-
+    this.registerConfigIfNotAlreadySet(
+      env,
+      EnvironmentConfig.PROOF_OF_WORK,
+      () => process.env.PROOF_OF_WORK == 'enabled'
```

Comment:
We usually use `true` or `false` with boolean variables
```suggestion
      () => process.env.PROOF_OF_WORK === 'true'
```